README.md

```diff
@@ -6,5 +6,15 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+Rails is a framework for creatign new projects in Ruby
+
 1. Using a marker, pen, or pencil, sketch the components of an (R)MVC architecture, how they interconnect, and which components have what responsibilities. Take a photo of your drawing and add it to the `submissions` directory inside your clone.
+
+
 1. Describe the flow of a PATCH request as it goes through the app.
+The client asks the server to create or add something.
+The server asks the appropriate controller.
+That controller parses the request to the proper model.
```

Comment:
The controller invokes methods on the models, causing the database to be manipulated in response to a request.